docs/manuals/user/dataMigrationFromSth-cometToQL/convert_json.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```

Comment:
May be a good idea to document which requirements users need to run this script (python2, 3, any pip install? etc). It's easy to deduce, but again, think of people not using python frequently.